modules/database/variables.tf

```diff
@@ -127,13 +150,9 @@ variable "diagnostic_setting_enabled_log_categories" {
 }
 
 variable "diagnostic_setting_enabled_metric_categories" {
-  description = "A list of log categories to be enabled for this diagnostic setting."
+  description = "A list of metric categories to be enabled for this diagnostic setting."
   type        = list(string)
-  default = [
-    "Basic",
-    "InstanceAndAppAdvanced",
-    "WorkloadManagement"
-  ]
+  default     = []
```

Comment:
Is this because metrics are set by default on the master database in the module?

Reply:
It's because we don't generally send metrics to Log Analytics by default.

Metrics are still available, they just won't be stored in Log Analytics.